redbot/cogs/economy/economy.py

```diff
@@ -989,23 +990,36 @@ async def rolepaydayamount(self, ctx: commands.Context, role: discord.Role, cred
         """
         guild = ctx.guild
         max_balance = await bank.get_max_balance(ctx.guild)
-        if creds <= 0 or creds > max_balance:
+        if creds >= max_balance:
             return await ctx.send(
-                _("Amount must be greater than zero and less than {maxbal}.").format(
-                    maxbal=humanize_number(max_balance)
-                )
+                _(
+                    "The bank requires that you set the payday to be less than it's maximum balance of {maxbal}."
```

Comment:
We try to stick to max line length in strings too, but Black doesn't automatically handle those yet.
```suggestion
                    "The bank requires that you set the payday to be less than"
                    " its maximum balance of {maxbal}."
```